### ISSUE TITLE
xlinux cmake build Failed to merge PDAT files

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#9869]
https://ci.eclipse.org/openj9/job/Build_JDK8_x86-64_linux_cm_OMR/624/
cent6-x64-5
```
16:19:49  stat failed /home/jenkins/workspace/Build_JDK8_x86-64_linux_cm_OMR/build/linux-x86_64-normal-server-release/vm/runtime/stFlSwjT
16:19:49  Failed to merge PDAT files
16:19:49  runtime/CMakeFiles/run_tracemerge.dir/build.make:99: recipe for target 'runtime/tracemerge.stamp' failed
16:19:49  gmake[4]: *** [runtime/tracemerge.stamp] Error 255
```

**Comment by pshipton** [comment]
@jdmpapin fyi.
Never seen this before, I restarted the build to see if it occurs again.

**Comment by dnakamura** [comment]
Is this also popping up in the jdk 11 builds?

**Comment by pshipton** [comment]
The reported failure is the first and only time I've seen this so far.

**Comment by dnakamura** [comment]
Looking at the tracemerge code, it looks like if it ever fails to `stat` a file while walking a directory tree it will fail. Given:
```
stat failed /home/jenkins/workspace/Build_JDK8_x86-64_linux_cm_OMR/build/linux-x86_64-normal-server-release/vm/runtime/stFlSwjT
```
It would appear that a temporary file was created in `vm/runtime`, but deleted between listing the directory and actually getting around to stat the file.

**Comment by pshipton** [comment]
Another one https://ci.eclipse.org/openj9/job/Build_JDK11_x86-64_linux_Personal/868

**Comment by keithc-ca** [comment]
This should be fixed by eclipse/omr#5328.
